UefiTestingPkg/AuditTests/PagingAudit/Windows/BinaryParsing.py

```diff
@@ -65,17 +65,17 @@ def Parse4kPages(fileName, addressbits, architecture):
             if 0 == (ByteArray[byteZeroIndex + 0] + ByteArray[byteZeroIndex + 1] + ByteArray[byteZeroIndex + 2] + ByteArray[byteZeroIndex + 3] + ByteArray[byteZeroIndex + 4] + ByteArray[byteZeroIndex + 5] + ByteArray[byteZeroIndex + 6] + ByteArray[byteZeroIndex + 7]):
                 byteZeroIndex += 8
                 continue
-            Valid = ((ByteArray[byteZeroIndex + 0] & 0x1))
+            AccessFlag = ((ByteArray[byteZeroIndex + 1] & 0x4) >> 2)
             IsTable = ((ByteArray[byteZeroIndex + 0] & 0x2) >> 1)
             AccessPermisions = (((ByteArray[byteZeroIndex + 0] & 0xC0) >> 6))
             Sharability = ((ByteArray[byteZeroIndex + 1] & 0x3))
-            Pxn         = ((ByteArray[byteZeroIndex + 6] & 0x10) >> 4)
-            Uxn         = ((ByteArray[byteZeroIndex + 6] & 0x20) >> 5)
+            Pxn         = ((ByteArray[byteZeroIndex + 6] & 0x20) >> 5)
+            Uxn         = ((ByteArray[byteZeroIndex + 6] & 0x40) >> 6)

```

Comment:
I take it these were reading the wrong bits before?

Reply:
Yes, the first commit in the rebase & ff is the bugfix